contracts/payment/SplitPayment.sol

```diff
@@ -36,27 +36,26 @@ contract SplitPayment {
   function () external payable {}
 
   /**
-   * @dev Claim your share of the balance.
+   * @dev Release one of the payee's proportional payment.
+   * @param _payee Whose payments will be released.
    */
-  function claim() public {
-    address payee = msg.sender;
-
-    require(shares[payee] > 0);
+  function release(address _payee) public {
```

Comment:
I think a plain `withdraw` is better than both of those, actually. I'm also not sure why we have `escrow.withdraw` but `pullPayment.withdrawPayments`.

Reply:
I think we've had this discussion before. The thing is `TokenTimelock` is using `release`. It would be nice to make it consistent and use the same term everywhere, if it makes sense.

Reply:
We probably had :stuck_out_tongue: I'd rather rename `TokenTimelock`'s to `withdraw`, but this is not the place for that discussion.